worker/common/hooks/testing/suite.go

```diff
@@ -0,0 +1,120 @@
+// Copyright 2017 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package testing
```

Comment:
can you please give the package name a bit more context? like "hookstesting". otherwise we'll end up having to alias it (or some other "testing" package) everywhere